config/jobs/kubernetes/kops/build_jobs.py

```diff
@@ -774,7 +774,7 @@ def generate_misc():
                    networking="cilium",
                    k8s_version="ci",
                    kops_channel="alpha",
-                   feature_flags=['SELinuxMount'],
+                   feature_flags=['SELinuxMount,SELinux'],
```

Comment:
```suggestion
                   extra_flags=[
                       '--set=cluster.spec.containerd.selinuxEnabled=true'
                   ],
```

Reply:
Tested with `kops create cluster --set=cluster.spec.containerd.selinuxEnabled=true ...`, works as expected.
I also regenerated all CI jobs, Ubuntu got new images.

Reply:
Wonderful! 😁

Reply:
@jsafrane I see in the end we got `"--set=Containerd.SeLinuxEnabled=true"`. Not quite sure if correct.

Reply:
```
I0621 01:41:08.891574    5070 local.go:42] ⚙️ /home/prow/go/src/k8s.io/kops/_rundir/a63cabe8-1e7b-42e7-a0b9-20bb608a01a9/kops create cluster --name e2e-e2e-kops-aws-selinux.test-cncf-aws.k8s.io --cloud aws --kubernetes-version https://storage.googleapis.com/k8s-release-dev/ci/v1.28.0-alpha.3.7+bbc7ca94a429c6 --ssh-public-key /tmp/kops/e2e-e2e-kops-aws-selinux.test-cncf-aws.k8s.io/id_ed25519.pub --set cluster.spec.nodePortAccess=0.0.0.0/0 --image=309956199498/RHEL-8.8.0_HVM-20230503-x86_64-54-Hourly2-GP2 --channel=alpha --networking=cilium --kubernetes-feature-gates=SELinuxMountReadWriteOncePod,ReadWriteOncePod --set=Containerd.SeLinuxEnabled=true --discovery-store=s3://k8s-kops-prow/discovery --admin-access 34.70.161.105/32 --master-count 1 --master-volume-size 48 --node-count 4 --node-volume-size 48 --zones ap-southeast-1b --master-size c5.large
I0621 01:41:08.922990    5114 featureflag.go:166] FeatureFlag "SELinuxMount"=true
I0621 01:41:08.966233    5114 create_cluster.go:882] Using SSH public key: /tmp/kops/e2e-e2e-kops-aws-selinux.test-cncf-aws.k8s.io/id_ed25519.pub
I0621 01:41:09.569684    5114 new_cluster.go:1351] Cloud Provider ID: "aws"

Error: field Containerd.SeLinuxEnabled not found in *Cluster
```